coala_quickstart/green_mode/Setting.py

```diff
@@ -0,0 +1,68 @@
+settings_key = 'green_mode_infinite_value_settings'
+
+
+def find_max_of_setting(setting, value, contents):
+    """
+    Generates max value of a setting where this
+    function is called upon for every value generated for
+    every file in the project (excluding ignored files).
+    :param setting:
+        The setting for which to find the max value of.
+    :param value:
+        The current value to be compared against the
+        supposedly max value stored in contents.
+    :param contents:
+        The python object to be written to 'PROJECT_DATA'
+        which contains the max value of the setting which was
+        encountered uptil now.
+    :return:
+        The contents with the max value of the setting encountered
+        uptil now after comparing it with the current value recieved
+        by the function.
+    """
+    found = False
+    position = None
+    for index, item in enumerate(contents[settings_key]):
+        if isinstance(item, dict) and setting in item:
+            found = True
+            position = index
+    if not found:
+        contents[settings_key].append({setting: value})
+        return contents
+    current_val = contents[settings_key][position][setting]
+    if value > current_val:
+        contents[settings_key][position][setting] = value
+    return contents
+
+
+def find_min_of_setting(setting, value, contents):
```

Comment:
the contents of this function look identical (except for one character) to the above ;-(